src/pfs/pfs.go

```diff
@@ -18,6 +18,8 @@ const (
 	UserRepoType = "user"
 	MetaRepoType = "meta"
 	SpecRepoType = "spec"
+
+	DefaultProjectName = "default"
```

Comment:
nit: NBD, but I would call it `DefaultProject`, feeling that the type implies it's the name.

Reply:
I started with `DefaultProject`, to be honest, but as I was using it I realised that it was a little less ergonomic than it could have been, because it is really a name rather than a project.